src/constants/scripts.ts

```diff
@@ -26,6 +26,372 @@ export const ScriptTagExtraRules = new Map<string, (s: Script) => string>([
   ],
 ])
 
+type ScriptAttributes = Record<'name' | 'description', string> &
+  Partial<Record<'code' | 'rfc' | 'deprecated' | 'website' | 'doc', string>>
+
+export const scripts = new Map<string, ScriptAttributes>([
```

Comment:
I though these data would be returned from the API as 

> We should uniform the source and use only api from the backend.

was mentioned in the issue(https://github.com/Magickbase/ckb-explorer-public-issues/issues/893)

By doing so, the label will be defined only in the database.

Reply:
In node connection mode, these data have been used.So I can't remove them.